detect_utm_zone.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+
+"""
+Accepts a polygon shapefile in WGS84 and finds the SRID for the UTM zone
```

Comment:
Re: "In WGS84". The current analysis scripts allow the neighborhood shapefile to be in any projection. If WGS84 is a limitation of this script, we should either ensure this script can handle any SRID, or remove the `NB_INPUT_SRID` option from the import scripts so that this doesn't cause confusion down the road.